src/components/CreateProjectForm.js

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable */
+import React, { useState } from 'react'
+import Grid from '@material-ui/core/Grid'
+import TextField from '@material-ui/core/TextField'
+import { ActiveButton } from './button/ActiveButton'
+import Web3 from 'web3'
+
+let web3 = new Web3(window.ethereum)
+let cloneABI = JSON.parse(process.env.REACT_APP_CLONE_ABI)
+let factoryABI = JSON.parse(process.env.REACT_APP_CLONE_FACTORY_ABI)
+let contractAddress = process.env.REACT_APP_RINKEBY_CONTRACT;
+// let contractAddress = '0x6C30403F7507C977d5d0a708d55938d2b57a77BD'
+
+const defaultValues = {
+  projectName: '',
+  projectAddress: '',
+  threshold: '',
+  finderWallet: '',
+  owners: '',
+}
+
+const defaultProjectDetails = {
+  projectName: '',
+  totalShares: '',
+  cloneAddress: null,
+}
+
+export const CreateProjectForm = () => {
+  const [formValues, setFormValues] = useState(defaultValues)
+  const handleInputChange = (e) => {
+    const { name, value } = e.target
+    setFormValues({
+      ...formValues,
+      [name]: value,
+    })
+  }
+
+  const checkValue = async () => {
+    let address = await projectDetails.cloneAddress
+    console.log(address)
+    if (address) {
+      let clone = new web3.eth.Contract(cloneABI, address)
+      let cloneValue = await web3.eth.getBalance(address, (err, res) => res)
+      var updatedDetails = {}
+      updatedDetails['cloneValue'] = cloneValue
+      console.log(cloneValue)
+      setProjectDetails({ ...projectDetails, ...updatedDetails })
+    }
+  }
+
+  const releaseFunds = async () => {
+    let address = await projectDetails.cloneAddress
+    console.log(address)
+    if (address) {
+      web3.eth.getAccounts().then(async (a) => {
```

Comment:
This might be tackled later (I would prefer sooner than later tho) but I think that the functions that interact directly with any third party service should be moved to another folder called `services` - This way we can create the interfaces (and implementations) of the methods that interact with the UI components, allowing us to componentize the different business logic of the app, hence, making the code more maintainable

If this doesn't makes sense just let me know and we can have a call :-D